src/MSXMotherBoard.cc

```diff
@@ -705,8 +705,8 @@ MSXMapperIO& MSXMotherBoard::createMapperIO()
 
 		MSXCPUInterface& cpuInterface = getCPUInterface();
 		for (auto port : {0xfc, 0xfd, 0xfe, 0xff}) {
```

Comment:
Even though values fit in a byte, compiler still complains about lose of precision.
```
src/MSXMotherBoard.cc:707:18: warning: implicit conversion loses integer precision: 'const int' to 'byte' (aka 'unsigned char') [-Wimplicit-int-conversion]
  707 |                 for (byte port : {0xfc, 0xfd, 0xfe, 0xff}) {
      |                           ~~~~ ^
```